src/main/java/in/projecteka/consentmanager/consent/ConsentServiceProperties.java

```diff
@@ -17,4 +17,5 @@
 public class ConsentServiceProperties {
     private int maxPageSize;
     private int defaultPageSize;
+    private String consentServiceUrl;
```

Comment:
```suggestion
    private String consentServiceUrl;
```
Can this be just an URL?

Reply:
Done